pilot/pkg/config/kube/gateway/deploymentcontroller.go

```diff
@@ -146,16 +142,6 @@ func getClassInfos() map[gateway.GatewayController]classInfo {
 		},
 	}
 
-	if features.MultiNetworkGatewayAPI {
```

Comment:
I am not sure I undertstand. The PR title is "add comments" but we are still deleting real code. Is this meant to be reverted?

Reply:
This seems not needed

Reply:
@stevenctl First added, can you give some insight

Reply:
This is for using Gateway API to specify a remote eastwest gateway without actually deploying things. 

If I have a WorkloadEntry with network: foo and a Gateway with network: foo, traffic to that "workload" goes through the gateway. Allows declaring external destinations without reading an external API server.

Reply:
IIUC, after removing all these stuff, and only keep `istio-remote` does not affect this feature

Reply:
You're right.. I can't figure out why this was needed.

Reply:
Np, we can remove safely now

Reply:
See John's comment. I was under the impression classes were static in the charts but they're generated here.